### ISSUE TITLE
StatusCode.PERMISSION_DENIED,  "Request had insufficient authentication scopes." during setup

### ISSUE THREAD

**Issue by ejbolt** [google/clusterfuzz#1464]
After running 'create_config' from butler.py, I run into the following error:

```
Traceback (most recent call last):
  File "butler.py", line 328, in <module>
    main()
  File "butler.py", line 315, in main
    command.execute(args)
  File "src/local/butler/run.py", line 45, in execute
    script.execute(args)
  File "src/local/butler/scripts/setup.py", line 304, in execute
    setup_config(args.non_dry_run)
  File "src/local/butler/scripts/setup.py", line 228, in setup_config
    config = data_types.Config.query().get()
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/query.py", line 1235, in wrapper
    return wrapped(self, *dummy_args, _options=query_options)
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/utils.py", line 78, in wrapper
    return wrapped(*args, **new_kwargs)
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/utils.py", line 110, in positional_wrapper
    return wrapped(*args, **kwds)
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/query.py", line 2115, in get
    return self.get_async(_options=kwargs["_options"]).result()
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/tasklets.py", line 190, in result
    self.check_success()
  File "/home/evan_bolt/clusterfuzz/src/third_party/google/cloud/ndb/tasklets.py", line 137, in check_success
    raise self._exception
grpc._channel._Rendezvous: <_Rendezvous of RPC that terminated with:
	status = StatusCode.PERMISSION_DENIED
	details = "Request had insufficient authentication scopes."
	debug_error_string = "{"created":"@1582156994.257066580","description":"Error received from peer","file":"src/core/lib/surface/call.cc","file_line":1099,"grpc_message":"Request had insufficient authentication scopes.","grpc_status":7}"
```
I've followed the setup steps here: https://google.github.io/clusterfuzz/production-setup/clusterfuzz/, I've enabled Firebase and added my domain to the whitelist, gotten a Firebase API Key, I've set up OAuth and have the .json secrets file on my server, and I've also defined all of the environment variables in the documentation and can confirm they're populated.  The app is even running at the domain, however I can't sign in with Google, and no email sign-in option appears, perhaps that is related?  Is there somewhere the setup gives more detailed logs?  Or any other information I can provide for troubleshooting?  I've followed the traceback and looked at the code at each location but I don't have enough of an understanding with the source to make sense of what the issue is.

**Comment by ejbolt** [comment]
Closing, fixed by following solution in [#242 ](https://github.com/google/clusterfuzz/issues/244)

**Comment by inferno-chromium** [comment]
Sorry for slow response, glad it worked. Were docs less clear or any way we can improve those.
